docs/libraries/libraries-with-cli.md

```diff
@@ -0,0 +1,563 @@
+# Creating Libraries on Any OS
+
+By [Phillip Carter](https://github.com/cartermp)
+
+**Some details are subject to change as the toolchain evolves.**
+
+Targeting .NET Core for new libraries can be done a number of ways, depending on the platform(s) you wish to support.  If you cannot or choose not to use Visual Studio to build and test new libraries or wish to use the command line tooling on any Operating System, then this guide is for you.
```

Comment:
Good point @richlander.  I think that although this is fairly focused on creating new libraries, its content is general-purpose enough to allow people to learn how to convert their existing libraries.